### PR TITLE
Fix image upload command for Windows and Linux

### PR DESCRIPTION
**Pull request by derskeal**

### CODE REVIEW

File: src/components/Props.jsx [hunk 1]
```diff
@@ -190,7 +197,7 @@ class Props extends Component {
                   <InputLabel className={classes.light} htmlFor='propRequired'>Required?</InputLabel>
                   <Switch
                     checked={this.state.propRequired}
-                    onChange={this.handleChange}
+                    onChange={this.togglePropRequired}
```
Comment (refinedblessing) [inline src/components/Props.jsx]: Nice catch!!!

File: src/components/Props.jsx [hunk 2]
```diff
@@ -77,13 +77,14 @@ const typeOptions = [
     key=''
   >
   </option>,
-  ...Object.keys(availablePropTypes).map(
-    type => <option
+  ...Object.keys(availablePropTypes).map(type =>
```
[the remark this answers is not in the document]
Comment (derskeal) [reply src/components/Props.jsx]: for some reason, I'm not getting any errors apart from a dependency issue.

**Comment by derskeal** [comment]
The package-lock.json deletion was a miss. But the `pre-start` command is exactly the same as the `build` command.

**Comment by refinedblessing** [comment]
Yeah, pre-start is same as build. But they are doing different things here. We have pre-start so that whenever you run yarn start, you do not have to run yarn build separately on another terminal especially if you are just trying to run the app and not developing it.

**Comment by derskeal** [comment]
Alright then.

File: src/components/Info.jsx [hunk 3]
```diff
@@ -1,8 +1,12 @@
-import React from 'react';
+import React from "react";
+
+let command;
+
+window.navigator.platform.match(/^mac/) ? command = 'Cmd+O' : command = 'Ctrl+Shift+O';
```
Comment (refinedblessing) [inline src/components/Info.jsx]: You could clean this up to be
const command = window.navigator.platform.match(/^mac/) ? 'Cmd+O' : 'Ctrl+Shift+O';

Comment (derskeal) [reply src/components/Info.jsx]: Oh my. Why am I in such a hurry?...

**Pull request merged**